integrations/crowdnode/src/main/res/layout/dialog_staking.xml

```diff
@@ -14,55 +14,248 @@
+                android:background="@drawable/round_corners_white_bg"
+                android:paddingBottom="35dp"
+                app:layout_constraintTop_toBottomOf="@id/staking_message_two"
+                tools:layout_editor_absoluteX="-9dp">
+
+                <TextView
+                    android:id="@+id/staking_benefits"
+                    style="@style/Caption.Medium"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginHorizontal="15dp"
+                    android:layout_marginTop="10dp"
+                    android:gravity="start"
+                    android:text="@string/crowdnode_staking_crowdnode_benefits_title"
+                    android:textAlignment="gravity"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toTopOf="parent" />
+
+                <ImageView
+                    android:id="@+id/staking_join_pool"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="15dp"
+                    android:src="@drawable/ic_join_the_pool"
+                    app:layout_constraintStart_toStartOf="@id/staking_benefits"
+                    app:layout_constraintTop_toBottomOf="@id/staking_benefits" />
+
+                <TextView
+                    android:id="@+id/stacking_join_pool_title"
+                    style="@style/Subtitle2"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="10dp"
+                    android:text="@string/crowdnode_staking_join_pool_title"
+                    app:layout_constraintBottom_toBottomOf="@+id/staking_join_pool"
+                    app:layout_constraintStart_toEndOf="@id/staking_join_pool"
+                    app:layout_constraintTop_toTopOf="@+id/staking_join_pool" />
+
+                <TextView
+                    android:id="@+id/staking_join_pool_message"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_marginEnd="15dp"
+                    android:text="@string/crowdnode_staking_join_pool_message"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="@id/stacking_join_pool_title"
+                    app:layout_constraintTop_toBottomOf="@id/stacking_join_pool_title" />
+
+                <ImageView
+                    android:id="@+id/staking_first_min_deposit"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="15dp"
+                    android:src="@drawable/ic_staking_first_deposit"
```

Comment:
This icon seems to have the wrong aligning. It ends up being aligned to the middle of the text block, while it should be slightly higher than the title.

Reply:
Somehow this got in there and changed the alignment. `app:layout_constraintVertical_bias="0.0"`

Reply:
fixed